src/xmpp/omemo.c

```diff
@@ -562,7 +569,9 @@ _omemo_bundle_publish_configure(xmpp_stanza_t* const stanza, void* const userdat
         log_error("[OMEMO] cannot configure bundle to an open access model");
         return 0;
     }
-    form_set_value(form, tag, "open");
+    // NOTE: The OMEMO XEP requires "pubsub#max_items=max", but this is not
+    // supported by Prosody, and this option is also not set in other clients
+    // like Gajimm.
```

Comment:
small typo in Gajim :)
maybe you can also ammend this commit to include the links to prosody that you added as comments to this PR